courses/views_test.py

```diff
@@ -422,6 +422,7 @@ def test_errory_status_code_returns_error(self):
         with mock.patch('courses.views.requests', autospec=True) as mock_req:
             mock_req.post.return_value.status_code = 500
             mock_req.post.return_value.content = 'some error text'
```

Comment:
We should stop mocking this if we're not using it, especially since it conflicts with the `json` return value below